internal/integration/generated.go

```diff
@@ -3078,7 +3086,63 @@ type simpleQueryResponse struct {
 // GetMe returns simpleQueryResponse.Me, and is useful for accessing the field via an interface.
 func (v *simpleQueryResponse) GetMe() simpleQueryMeUser { return v.Me }
 
-// The query or mutation executed by createUser.
+// The query, mutation or subscription executed by count.
+const count_Operation = `
+subscription count {
+	count
+}
+`
+
+func count(
+	ctx_ context.Context,
+	client_ graphql.Client,
+) (chan countWsResponse, chan struct{}, chan error, map[string]interface{}, error) {
```

Comment:
peut etre utiliser des named param pour expliciter a quoi correspond le `chan struct{}`
```suggestion
) (dataChan chan countWsResponse, doneChan chan struct{}, errChan chan error, map[string]interface{}, err error) {
```

Reply:
et pourquoi ne pas plutot faire un `chan bool` plutot que `chan struct{}` ? 
Ca serait plus simple a comprendre quand on ferme la chan de faire:
```
defer func() { doneChan <- True }()
```
plutot que 
```
defer func() { doneChan <- struct{}{} }()
```

Reply:
done